### DISCUSSION TITLE
[Kavita] Export manually created reading list as cbl file

### DISCUSSION THREAD

**Discussion by jncmney** [Kareadita/Kavita#3227] [Ideas]
### Idea Description

Kavita can read/import reading lists from cbl files. It would be great to be able to export a manually build reading list as a fully formatted cbl file well. 

Two reasons:

1. Sharing with others. 
2. Back up. 

One example, there’s some debate about what is considered canon Star Wars. There are lists for only the Marvel series or only the DH series, but I cannot find one that combines the “canon” series of both. I manually built one that I’d be happy to share. 

### Idea Category

Feature Enhancement

### Duration of Using Kavita

1 year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by nbycoder** [comment]
This would be super useful for me too. Especially since I use Linux and I don't have access to Comic Rack anymore.

**Comment by YumDumbPickle** [comment]


**Reply by majora2007** [reply]
You don't need to bump anything, you upvote the request.
